scripts/test-sim-versions.go

```diff
@@ -0,0 +1,132 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+// +build ignore
+
+package main
+
+import (
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/exec"
+	"strings"
+
+	"gopkg.in/yaml.v2"
+)
+
+var SkippedUplinkVersions = []string{
```

Comment:
can we add a comment above this explaining why we skip these versions?

Reply:
I know we're not testing ones before version 15 but aren't there more versions before "v0.10.1"? is there a reason those aren't included in this list?